src/runtime_src/core/edge/user/system_linux.cpp

```diff
@@ -85,7 +100,20 @@ glibc_info()
   return _pt;
 }
 
-void 
+static std::string
+machine_info()
+{
+  static const char node[] = MACHINE_NODE_PATH;
```

Comment:
What's the point of node here? Why not use MACHINE_NODE_PATH directly line 108?

Reply:
Done, see above.